src/modules/search/SearchResults.tsx

```diff
@@ -15,15 +20,32 @@ export interface SearchResultsProps {
 
 const SearchResults: React.FC<SearchResultsProps> = ({ resources, searchTerm }) => {
   const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const handleSearch = (offset: number) => {
+    if (searchTerm.length > 0) {
+      dispatch(search(searchTerm, offset));
+      history.push(`/Search/${searchTerm}`);
+    }
+  };
+
+  const results = useSelector((state: RootStore) => state.results);
+  const [offset, setOffset] = useState(0);
+
+  const updateOffset = (offset: number) => {
```

Comment:
egentlig ikke nødvendig å ha en egen funksjon for å oppdatere offset. Kan gjøre det i handleSearch/updateSearch?